apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/ChatModel.kt

```diff
@@ -65,6 +66,7 @@ object ChatModel {
   // current chat
   val chatId = mutableStateOf<String?>(null)
   val chatItems = mutableStateOf(SnapshotStateList<ChatItem>())
+  val chatItemsSectionArea = mutableMapOf<Long, ChatSectionArea>()
```

Comment:
make a comment about what Long parameter means, its not obvious based on context